src/hotspot/os_cpu/linux_riscv/riscv_hwprobe.cpp

```diff
@@ -178,6 +178,9 @@ void RiscvHwprobe::add_features_from_query_result() {
   if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZFH)) {
     VM_Version::ext_Zfh.enable_feature();
   }
+  if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZVBB)) {
+    VM_Version::ext_Zvbb.enable_feature();
```

Comment:
Same as https://github.com/openjdk/jdk/pull/19065, we don't want to enable experimental extensions via hwprobe.

Reply:
Same as above. Thanks!

Reply:
That is still leftover from merging master.